diff --git a/node_modules/@vuedoc/parser/lib/parser/EventParser.js b/node_modules/@vuedoc/parser/lib/parser/EventParser.js
index 872f03e..3a3634a 100644
--- a/node_modules/@vuedoc/parser/lib/parser/EventParser.js
+++ b/node_modules/@vuedoc/parser/lib/parser/EventParser.js
@@ -23,7 +23,7 @@ class EventParser extends AbstractExpressionParser {
     switch (node.type) {
       case Syntax.Identifier:
         return this.scope.hasOwnProperty(node.name)
-          ? this.scope[node.name]
+          ? (this.scope[node.name] || UnhandledValue)
           : UnhandledValue;
 
       case Syntax.ObjectExpression:
